block/pkg/datastore/common/utils.go

```diff
@@ -0,0 +1,36 @@
+// Copyright 2020 The OpenSDS Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package common
+
+import (
+	_ "context"
+	//"github.com/opensds/multi-cloud/block/pkg/datastore/driver"
+)
+
+func TrimQuot(in string) string {
```

Comment:
What s the usage of this function?

Reply:
Removed..had copied and kept if required in future